orchestration/hca_orchestration/config/prod_migration/run_config/dcp1_real_prod_migration.yaml

```diff
@@ -0,0 +1,22 @@
+resources:
+  hca_project_copying_config:
+    config:
+      source_bigquery_project_id: broad-datarepo-terra-prod-hca2
+      source_snapshot_name: hca_prod_20201118_dcp1___20201209
+      source_bigquery_region: US
+  load_tag:
+    config:
+      append_run_id: true
+      load_tag_prefix: prod_migration
+  scratch_config:
+    config:
+      scratch_bq_project: NA
+      scratch_bucket_name: broad-dsp-monster-prod-migration-temp-uc1
```

Comment:
This is a "us-central1" bucket as both DCP1 and DCP2 snapshots reside in us-central1. This will be less needed when we switch back to direct copy to TDR in a follow-on PR.